.github/workflows/lint-test.yaml

```diff
@@ -167,8 +167,9 @@ jobs:
       - uses: taiki-e/install-action@v1.5.6
         with:
           tool: nextest
+      # prebuild node binary, so that tests don't start before node is built.
       # --test-threads is intentionally set to reduce resource contention in ci jobs. Increasing this, increases job failures and retries.
-      - run: cargo nextest run --profile ci --package smoke-test --test-threads 6 --retries 3
+      - run: cargo build --bin=aptos-node --features=failpoints --release && LOCAL_SWARM_NODE_RELEASE=1 cargo nextest run --release --profile ci --package smoke-test --test-threads 6 --retries 3
```

Comment:
we probably can split the build into a separate step?

Reply:
can it be a separate step? How do I make sure it is running on the same machine, and output of that step is there when I start this step?

Reply:
plus if someone copies the command to re-run smoke tests locally, they are going to have a better experience with running the full command

Reply:
I think all steps here are run on the same machine, just need another line of -run? but not sure